packages/neuron-wallet/src/controllers/networks/index.ts

```diff
@@ -26,7 +24,7 @@ export default class NetworksController {
           await this.connectToNetwork(true)
         } else {
           logger.debug('Network:\tconnection dropped')
-          resetSyncTaskQueue.asyncPush(false)
+          resetSyncTaskQueue.push(false)
```

Comment:
Why change `asyncPush` to `push`?

Reply:
<img width="830" alt="image" src="https://github.com/nervosnetwork/neuron/assets/11756869/22c0af25-e5e7-478c-8c33-f83a045c0966">

Reply:
After this https://github.com/nervosnetwork/neuron/commit/1dfd25543ec381a0831d297039f84a56dc22ebc5#diff-71d0fc5f2524051573afeb6aa962ebc8277a5feb532501bf2f441855e07e7a33R29, it changed to `asyncPush`

Reply:
My mistake, I amend